paket.dependencies

```diff
@@ -3,6 +3,7 @@ source https://nuget.org/api/v2
 nuget Newtonsoft.Json redirects: on
 nuget Argu
 nuget FSharp.Core redirects: on
+nuget NuGet.Versioning
```

Comment:
ah "Adds a reference to NuGet.Versioning so that we can reuse there parsing logic for the new version range syntax."

I don't think that's a good idea. we already have parse logic and I think we should take that dependency. We're much faster with fixes if this little parse code is under our control.